README.md

```diff
@@ -1,2 +1,197 @@
-# To-Do-React
-A To Do app created with React is a web application that allows users to create, manage and organize their daily tasks or to-do lists using an intuitive and responsive interface.
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h1><b>To Do React</b></h1>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Math Magicians React <a name="about-project"></a>
+
+
+**To Do React** is created with React is a web application that allows users to create, manage and organize their daily tasks or to-do lists using an intuitive and responsive interface..
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS/SASS</li>
+    <li>JavaScript</li>
+    <li>React</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>GitHub Pages</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>No database was required for this project</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[This project use React]**
+- **[This project does a correct use of GitHub Flows]**
+
+<p align="right"><a href="#readme-top">Back to top</a></p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[To Do React - Live Demo](https://nicolasandrehh.github.io/To-Do-React/)
```

Comment:
**The live Demo is not working, remember to fix using: [Part 11 – Deploying React app to GitHub Pages](https://ibaslogic.com/deploying-react-app-to-github-pages/)**